src/cpp/core/r_util/RToolsInfo.cpp

```diff
@@ -206,10 +241,23 @@ RToolsInfo::RToolsInfo(const std::string& name,
 
 std::string RToolsInfo::url(const std::string& repos) const
 {
-   // strip period from name
-   std::string ver = boost::algorithm::replace_all_copy(name(), ".", "");
-   std::string url = core::http::URL::complete(
-                        repos, "bin/windows/Rtools/Rtools" + ver + ".exe");
+   std::string url;
+
+   if (name() == "4.0")
+   {
+      // TODO: Currently, Rtools 4.0 is only available from the 'testing'
+      // sub-directory. Update this URL once it's been promoted.
```

Comment:
Is this promotion likely to happen in concert with the 4.0 release? (wondering if we should go ahead and merge this URL for 1.3 and then update to the final URL in the 1.3 patch release)

Reply:
Yes, it should happen either in tandem with or shortly before the 4.0.0 release.